lib/mock-component/mock-component.spec.ts

```diff
@@ -56,33 +63,33 @@ describe('MockComponent', () => {
   it('should have the input set on the mock component', () => {
     fixture.detectChanges();
     const mockedComponent = fixture.debugElement
-                                   .query(By.directive(MockComponent(SimpleComponent)))
-                                   .componentInstance;
+      .query(By.directive(MockComponent(SimpleComponent)))
+      .componentInstance;
```

Comment:
Why all the formatting?  It is making it inconsistent with the rest of the app.

Reply:
Sorry, this is just Webstorm, or prettier auto formatting based on whatever settings it has found. I have tried to fix this up. If it's still an issue let me know what editor you use and I will try to open and reformat in that.